test/integration/spec/datatrackPublish.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
```

Comment:
@makarandp0 Isn't there already a test for this:
```
#publishTrack
  called with an enabled
    LocalAudioTrack
      that has never been published
        should raise a "trackPublished" event on the corresponding RemoteParticipant with a RemoteTrackPublication
```
The `defaults` already has `dominantSpeaker: true`.

Reply:
@manjeshbhargav - This test is specifically for data tracks publishing failures. I have seen it has something to do with MSP objects (hence `[true, false].forEach((dominantSpeaker) `) and also is timing sensitive and thus `[0, 5000].forEach((delay) `

Reply:
@makarandp0 For the LocalDataTrack case, there is this scenario:
```
#publishTrack
  called with an enabled
    LocalDataTrack
      that has never been published
        should raise a "trackPublished" event on the corresponding RemoteParticipant with a RemoteTrackPublication
```
Also, I'm guessing that the `0` delay scenario is the problematic one? In that case, the above scenario should cover it no?

Reply:
@manjeshbhargav - I saw that adding a  delay of 5000 ms between Alice and bob connecting reproduces issue more frequently. I am hoping that we would know more once somebody investigates VMS-2202  and readjust the tests accordingly.